nbdev/quarto.py

```diff
@@ -259,14 +260,66 @@ def nbdev_readme(
     "Create README.md from readme_nb (index.ipynb by default)"
     cfg = get_config()
     path = Path(path) if path else cfg.nbs_path
-    if chk_time and _readme_mtime_not_older(cfg.config_path/'README.md', path/cfg.readme_nb): return
+    if chk_time and _doc_mtime_not_older(cfg.config_path/'README.md', path/cfg.readme_nb): return
 
     with _SidebarYmlRemoved(path): # to avoid rendering whole website
         cache = proc_nbs(path)
         _sprun(f'cd "{cache}" && quarto render "{cache/cfg.readme_nb}" -o README.md -t gfm --no-execute')
         
     _save_cached_readme(cache, cfg)
 
+# %% ../nbs/api/14_quarto.ipynb
+def _save_cached_contributing(cache, cfg, contrib_nb):
+    "Move CONTRIBUTING.md (and any `_files` assets) from the Quarto build cache to the repo root."
+    tmp_doc_path = cache / cfg.doc_path.name
+    contrib_file = tmp_doc_path / 'CONTRIBUTING.md'
+    if contrib_file.exists():
+        final_path = cfg.config_path / 'CONTRIBUTING.md'
+        if final_path.exists(): final_path.unlink()  # Py3.7 doesn't have missing_ok
+        move(contrib_file, final_path)
+        # Move any supporting files folder
+        assets_folder = tmp_doc_path / (Path(contrib_nb).stem + '_files')
+        if assets_folder.exists():
+            _copytree(assets_folder, cfg.config_path / assets_folder.name)
+
+# %% ../nbs/api/14_quarto.ipynb
+@call_parse
+def nbdev_contributing(
+    path:str=None,  # Path to notebooks (defaults to nbs_path)
+    chk_time:bool=False  # Only build if out-of-date
+):
+    """
+    Create CONTRIBUTING.md from contributing_nb (defaults to 'contributing.ipynb' if present).
+    Skips if the file doesn't exist.
+    """
+    cfg = get_config()
+    path = Path(path) if path else cfg.nbs_path
+    
+    # Decide which notebook is your "contributing" NB (you can hardcode or add to settings.ini)
+    contrib_nb_name = cfg.get('contributing_nb', 'contributing.ipynb')
+    contrib_nb_path = path / contrib_nb_name
+    
+    contrib_md = cfg.config_path / 'CONTRIBUTING.md'
+    
+    # If out of date check is requested, skip if up-to-date or missing
+    if chk_time and _doc_mtime_not_older(contrib_md, contrib_nb_path):
+        return
+    
+    # If there's no contributing notebook, do nothing
```

Comment:
comments should not just repeat what the code already does - only add comments where there's something surprising or hard to understand